### PR TITLE
Move engine tests

### PR DESCRIPTION
**Pull request by adriangonz**
## Changelog
- Move PR tests for engine to run within `/test engine`

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Tue Jul  7 09:18:31 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=1

**Comment by seldondev** [comment]
Tue Jul  7 09:18:40 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=2

**Comment by seldondev** [comment]
Tue Jul  7 09:28:02 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=3

**Comment by seldondev** [comment]
Tue Jul  7 09:28:16 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=4

File: jenkins-x.yml [hunk 1]
```diff
@@ -2,97 +2,89 @@ buildPack: none
 pipelineConfig:
   pipelines:
     overrides:
-    - name: changelog
-      pipeline: release
-      stage: promote
-      step:
-        command: echo "skipping promote"
+      - name: changelog
```
Comment (axsaucedo) [inline jenkins-x.yml:5]: Not sure the indent is necessary, was this auto-formatted?

File: jenkins-x.yml [hunk 2]
```diff
@@ -2,97 +2,89 @@ buildPack: none
...
-          - agent:
-              image: gcr.io/jenkinsxio/builder-go:2.0.916-264
-            dir: ci
-            sh: "./add-pr-build-comment"
-        - name: build-and-test
-          parallel:
-          - name: test-python
-            agent:
-              image: seldonio/python-builder:0.2
+          - name: pr-build-comment
```
Comment (axsaucedo) [inline jenkins-x.yml:15]: nit: same here overindent

**Comment by axsaucedo** [comment]
Looks good, did you add the change to the jx config to run this file?

File: jenkins-x.yml (hunk 1 above)
Comment (adriangonz) [reply jenkins-x.yml:5]: It was auto-formatted. Let me disable that and re-add the changes.

**Comment by adriangonz** [comment]
@axsaucedo I just re-added the engine trigger now.

**Comment by seldondev** [comment]
Tue Jul  7 12:33:17 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=5

**Comment by seldondev** [comment]
Tue Jul  7 12:34:24 UTC 2020
The logs for [engine] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=7

**Comment by seldondev** [comment]
Tue Jul  7 12:34:33 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=6

**Comment by axsaucedo** [comment]
Oh right @adriangonz , you actually have to add it here: https://github.com/SeldonIO/environment-jx-production-cluster-dev/blob/master/env/templates/seldon-core-scheduler.yaml

**Comment by seldondev** [comment]
Tue Jul  7 14:23:57 UTC 2020
The logs for [engine] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2082/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2082 --build=8

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/2082#pullrequestreview-443970836 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
failed to trigger Pull Request pipeline
* failed to create agent
* failed to calculate in repo config
* failed to load trigger config for repository SeldonIO/seldon-core for ref adf06233f81ec50ef418d6b4be58677d1c1b9e85
* failed to find any lighthouse configuration files in repo SeldonIO/seldon-core at sha adf06233f81ec50ef418d6b4be58677d1c1b9e85
* failed to process repo SeldonIO/seldon-core refref adf06233f81ec50ef418d6b4be58677d1c1b9e85
* failed to list files in directory /var/tmp/gitrepo660600678/.lighthouse
* open /var/tmp/gitrepo660600678/.lighthouse
* no such file or directory

**Pull request merged**